src/Serilog.HttpClient/RequestLoggingOptions.cs

```diff
@@ -13,11 +13,11 @@ namespace Serilog.HttpClient
     public class RequestLoggingOptions
     {
         const string DefaultRequestCompletionMessageTemplate =
-            "HTTP Client Request Completed {@Context}";
+            "HTTP {RequestMethod} {RequestUri} responded {StatusCode} in {ElapsedMilliseconds:0.0000} ms";
 
         /// <summary>
         /// Gets or sets the message template. The default value is
-        /// <c>"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms"</c>.
+        /// <c>"HTTP {RequestMethod} {RequestUri} responded {StatusCode} in {ElapsedMilliseconds:0.0000} ms"</c>.
```

Comment:
Now the default matches what the docs say. FWIW, this wouldn't have worked because the properties like RequestMethod weren't directly accessible.